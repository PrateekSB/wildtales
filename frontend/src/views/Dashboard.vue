<template>
  <div>
    <p>dashboard</p>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
import Header from "@/components/Header.vue";
import Filters from "@/components/Filters.vue";
import DataTable from "@/components/DataTable.vue";
import { action } from "vuex-class-component";
import { storeProxy, store } from "../store";

@Component({
  components: {
    Filters,
    DataTable,
    Header
  }
})
export default class Dashboard extends Vue {}
</script>

<style lang="scss" scoped>
.dashboard {
  display: flex;
  flex-direction: column;
}
</style>
