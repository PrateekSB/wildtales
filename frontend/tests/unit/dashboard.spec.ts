import { shallowMount } from '@vue/test-utils';
import Dashboard from '@/views/Dashboard.vue';

describe('Dashboard.vue', () => {
  test('renders dashboard component', () => {
    const wrapper = shallowMount(Dashboard);
    expect(wrapper.element).toMatchSnapshot();
  });
});
