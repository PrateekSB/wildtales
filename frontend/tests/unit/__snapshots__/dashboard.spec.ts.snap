// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard.vue renders dashboard component 1`] = `
<div>
  <p>
    dashboard
  </p>
</div>
`;
